subsys/usb/Kconfig

```diff
@@ -16,6 +17,24 @@ menuconfig USB_DEVICE_STACK
 
 if USB_DEVICE_STACK
 
+menuconfig USB_DEVICE_STACK_SETTINGS
+	bool
+	prompt "USB device stack settings"
+	default n
+	help
+	Allow to set USB device stack settings including Vendor ID
+	and Product ID.
+
+if USB_DEVICE_STACK_SETTINGS
+
+config USB_DEVICE_VENDOR_ID
+	hex "USB Device Vendor ID"
+
+config USB_DEVICE_PRODUCT_ID
+	hex "USB Device Product ID"
```

Comment:
Configurable VID and PID without the ability to configure manufacturer / product strings is not really practical.

Reply:
We can add later other values